salt/modules/htpasswd.py

```diff
@@ -139,3 +139,46 @@ def userdel(pwfile, user, runas=None):
     out = __salt__['cmd.run'](cmd, runas=runas,
                               python_shell=False).splitlines()
     return out
+
+
+def verify(pwfile, user, password, opts='', runas=None):
+    '''
+    Return True if the htpasswd file exists, the user has an entry, and their
+    password matches.
+
+    pwfile
+        Path to htpasswd file
```

Comment:
We should specify that this must be the fully-qualified path to the file, just so there is no confusion. Sometimes people try to assume a relative location. (Yes, I know that's a little weird but it happens.) ;]